packages/thumbprint-react/components/Image/index.jsx

```diff
@@ -261,6 +270,16 @@ Image.propTypes = {
      * The placeholder prevents the browser scroll from jumping when the image is lazy-loaded.
      */
     containerAspectRatio: PropTypes.number,
+    /**
+     * Overrides the default calculation of the `sizes` attribute. Generally only useful for
+     * images that are server-side rendered. See https://mzl.la/2Hh6neO for allowable values.
```

Comment:
```suggestion
     * images that must be loaded before JavaScript is parsed and executed client-side. This 
     * is often paired with the `disableLazyLoading` prop. 
     * [See allowable values](https://mzl.la/2Hh6neO).
```

Changing the wording a bit to clarify "images that are server-side rendered".